signalfx-tracing/signalfx-python-tracing/README.md

```diff
@@ -0,0 +1,11 @@
+# SignalFX Tracing Library for Python 
+
+This repository contains an example application that utilizes auto-instrumentation for 
+supported python web frameworks and clients. <br/>
+
+For more detailed information about the functionality of the provided tracer and instrumentation configuration, 
+please see the [`signalfx-tracing`](https://github.com/signalfx/signalfx-python-tracing/tree/master/scripts) scripts.
```

Comment:
I'd refer to the top level readme instead of the scripts directly.

Reply:
OK

Reply:
Checked the link again, it links to the README of the `sfx-py-trace-bootstrap` and `sfx-py-trace` scripts (SignalFx-Tracing Library for Python Utilities), isn't that helpful?